.travis.yml

```diff
@@ -88,23 +103,20 @@ matrix:
         - REPLICATION=Y
         - COVERAGE=Y
     - jdk: oraclejdk8
+      sudo: required
+      addons:
       addons:
-        postgresql: "9.4"
         apt:
           packages:
-            - oracle-java8-installer
-      sudo: required
-      dist: trusty
+            - oracle-java8-installer # Test with latest update
```

Comment:
That is an interesting move.

Would you add an environment variable so one can "ctrl+f" Travis results page and tell which is "plain Java8" and which is "Java8 with updates"?